db/data/regions/VN.yml

```diff
@@ -23,7 +23,7 @@ combined_address_format:
   address2:
     - key: line2
     - key: neighborhood
-      decorator: ", Quận"
```

Comment:
reflected the key decision to only use punctuation / special characters; no words